### ISSUE TITLE
gradient explosion[BUG]: 

### ISSUE THREAD

**Issue by ShiJiawenwen** [hpcaitech/ColossalAI#3316]
### 🐛 Describe the bug

Hi, thanks for your great work.
I run the train_reward_model.py with the model of "gpt-2(s)", strategy with "naive", but got a problem that loss = NaN, can you help me?

**Comment by ShiJiawenwen** [comment]
I also run the examples/train_rm.sh, still face the same problem

**Comment by JThh** [comment]
What was your command?

**Comment by ShiJiawenwen** [comment]
set_n_least_used_CUDA_VISIBLE_DEVICES 0

python train_reward_model.py --pretrain 'microsoft/deberta-v3-large' \
                             --model 'deberta' \
                             --strategy naive \
                             --loss_fn 'log_exp'\
                             --save_path 'rmstatic.pt' \
                             --test True

**Comment by ShiJiawenwen** [comment]
I try the model of gpt2 and deberta, the same problem
